tests/eiler_tests.py

```diff
@@ -0,0 +1,36 @@
+import unittest
+
+import ddt
+
+from src.Homework6 import eiler
+
+
+@ddt.ddt
+class TestHomework(unittest.TestCase):
+
+    @ddt.data(
+        (100, 1752),
+        (10, 5),
+        (200, 70516)
+    )
+    @ddt.unpack
+    def test_eiler(self, num, expected):
+        result = eiler.func(num)
+        self.assertEqual(result, expected)
+
+    @ddt.data(
+        -10,
+        0,
+    )
+    def test_invalid_input(self, num):
```

Comment:
missed docstrings. 
what is the difference between this and next? bad test namings